extensions/amp-story/0.1/amp-story-auto-ads.js

```diff
@@ -157,12 +212,14 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
    * @private
    */
   createAdElement_() {
-    // TODO(ccordry) get this info (e.g. source) from config
+    const {'data-src': dataUrl, type} = this.config_['ad-attributes'];
```

Comment:
we want to blindly copied over all attributes over.

Reply:
done. NOTE: I changed `data-src` to `data-url`. I think the intention was to make these keywords the same. Right now the spec says `data-src` while `amp-ad` custom expects `data-url`. I am assuming we should update spec

Reply:
yep, that was a mistake in the spec